invent-code-0/main.cpp

```diff
@@ -4,4 +4,11 @@ using namespace std;
 
 int main() {
   // Write code here.
+  cout << endl;
+  cout << "I graduated from TAMS in 2004." << endl;
+  cout << endl;
```

Comment:
it looks like you're duplicating your endl here. you have two new lines per one line of code. Can you remove one of the endls? here and below too.

Reply:
Fixed.